assets/xml/objects/object_kz.xml

```diff
@@ -1,93 +1,106 @@
-        <Animation Name="object_kz_Anim_0058B8" Offset="0x58B8" /> <!-- Original name is "kz_run" -->
+        <Animation Name="gKaizokuSlashAnim" Offset="0x3CC" /> <!-- Original name is "kz_attackA" -->
+        <Animation Name="gKaizokuSpinAttackAnim" Offset="0xF5C" /> <!-- Original name is "kz_attackB" -->
+        <Animation Name="gKaizokuFlipAnim" Offset="0x1390" /> <!-- Original name is "kz_back_ten" -->
+        <Animation Name="gKaizokuBlockAnim" Offset="0x1578" /> <!-- Original name is "kz_defense" -->
+        <Animation Name="gKaizokuThrowFlashAnim" Offset="0x1E9C" /> <!-- Original name is "kz_doron" -->
+        <!-- This is the old version from OOT, back when she would jump to escape instead of talking and flashing to escape -->
+        <Animation Name="gKaizokuUnusedDefeatAnim" Offset="0x2730" /> <!-- Original name is "kz_down" -->
+        <Animation Name="gKaizokuDefeatIdleAnim" Offset="0x2BA0" /> <!-- Original name is "kz_downwait" -->
+        <Animation Name="gKaizokuDefeatAnim" Offset="0x3A3C" /> <!-- Original name is "kz_downwake" -->
+        <!-- This is used in a threatening stance if the player loses the fight, while she talks to you -->
+        <Animation Name="gKaizokuDemonstrativeSwordSwingAnim" Offset="0x43E4" /> <!-- Original name is "kz_getout" -->
+        <Animation Name="gKaizokuDamageAnim" Offset="0x4860" /> <!-- Original name is "kz_hit" -->
+        <Animation Name="gKaizokuUnsheatheAnim" Offset="0x5644" /> <!-- Original name is "kz_kime" -->
+        <Animation Name="gKaizokuWalkAnim" Offset="0x58B8" /> <!-- Original name is "kz_run" -->
 
-        <Texture Name="object_kz_TLUT_0058D0" OutName="tlut_0058D0" Format="rgba16" Width="16" Height="16" Offset="0x58D0" />
-        <Texture Name="object_kz_Tex_005AD0" OutName="tex_005AD0" Format="ci8" Width="8" Height="8" Offset="0x5AD0" />
-        <Texture Name="object_kz_Tex_005B10" OutName="tex_005B10" Format="ci8" Width="8" Height="8" Offset="0x5B10" />
-        <Texture Name="object_kz_Tex_005B50" OutName="tex_005B50" Format="ci8" Width="16" Height="16" Offset="0x5B50" />
-        <Texture Name="object_kz_Tex_005C50" OutName="tex_005C50" Format="i8" Width="16" Height="16" Offset="0x5C50" />
-        <Texture Name="object_kz_Tex_005D50" OutName="tex_005D50" Format="ci8" Width="16" Height="16" Offset="0x5D50" />
-        <Texture Name="object_kz_Tex_005E50" OutName="tex_005E50" Format="i8" Width="16" Height="16" Offset="0x5E50" />
-        <Texture Name="object_kz_Tex_005F50" OutName="tex_005F50" Format="i8" Width="8" Height="16" Offset="0x5F50" />
+        <Texture Name="gKaizoku1TLUT" OutName="kaizoku_tlut1" Format="rgba16" Width="16" Height="16" Offset="0x58D0" />
+        <!-- This is the same skin texture as below, but this one is her arms and back -->
+        <Texture Name="gKaizokuSkinGradient1Tex" OutName="skin1" Format="ci8" Width="8" Height="8" Offset="0x5AD0" TlutOffset="0x58D0" />
```

Comment:
tbh, I've never been a big fan of SkinGradient since I think `Skin` by itself is sufficient.